uclibc does not have exp10 function which is glibc extension.
Bur we can get the same behavior by using pow()

Upstream-Status: Pending

Khem Raj <raj.khem@gmail.com>

Index: alsa-utils-1.0.24.2/alsamixer/volume_mapping.c
===================================================================
--- alsa-utils-1.0.24.2.orig/alsamixer/volume_mapping.c
+++ alsa-utils-1.0.24.2/alsamixer/volume_mapping.c
@@ -36,6 +36,9 @@
 #include <math.h>
 #include <stdbool.h>
 #include "volume_mapping.h"
+#ifdef __UCLIBC__
+#define exp10(x) (pow(10, (x)))
+#endif /* __UCLIBC__ */
 
 #define MAX_LINEAR_DB_SCALE	24
 
